lua/nvim-treesitter/parsers.lua

```diff
@@ -416,6 +416,15 @@ list.turtle = {
   maintainers = { "@bonabeavis" },
 }
 
+list.devicetree = {
+  install_info = {
+    url = "https://github.com/joelspadin/tree-sitter-devicetree",
+    files = { "src/parser.c" },
+    branch = "main",
+  },
+  filetype = 'dts',
+}
+
```

Comment:
So you will not maintain the queries for this language ?

Reply:
Probably I could